torch_xla/runtime.py

```diff
@@ -215,7 +215,7 @@ def global_runtime_device_attributes() -> List[Dict[str, object]]:
 @requires_pjrt
 @functools.lru_cache()
 def global_runtime_device_count() -> int:
-  """Returns the total number of runtime devices across all processes/hosts."""
+  """Returns the total number of runtime devices across all processes/hosts. Currently this is only used in SPMD."""
```

Comment:
I would say this is _useful_ for SPMD, since otherwise it's the same as `global_device_count`.

Reply:
done